src/handlers/auth.js

```diff
@@ -61,7 +61,13 @@ let signUp = async (req, res) => {
 
     let token = JWT.issueToken(signUpRes.user.uid, signUpRes.user.admin);
 
-    return res.status(201).send({user: signUpRes.user, token: token});
+    return res.status(201).send({user: {
+        uid: signUpRes.user.uid,
+        firstName: signUpRes.user.first_name,
+        lastName: signUpRes.user.last_name,
+        email: signUpRes.user.email,
+        admin: signUpRes.user.admin
+    }, token: token});
```

Comment:
Oh!! This makes sense why there was an inconsistent naming convention. Gotta love SQL 😂

Reply:
Haha oh yeah...